src/quo2/foundations/colors.cljs

```diff
@@ -272,24 +272,32 @@
      ([color suffix]
       (custom-color color suffix nil))
      ([color suffix opacity]
-      (let [hex?  (not (keyword? color))
-            color (if (hex-string? (get colors-map color))
-                    (get colors-map color)
-                    (get-in colors-map [color suffix]))]
+      (let [hex?           (not (keyword? color))
+            resolved-color (cond hex?                                 color
+                                 (hex-string? (get colors-map color)) (get colors-map color)
+                                 :else                                (get-in colors-map
+                                                                              [color suffix]))]
         (cond
-          (and opacity hex?) (alpha color (/ opacity 100))
-          opacity            (alpha color (/ opacity 100))
-          hex?               color
-          :else              color))))))
+          (and opacity hex?) (alpha resolved-color (/ opacity 100))
+          opacity            (alpha resolved-color (/ opacity 100))
+          hex?               resolved-color
+          :else              resolved-color))))))
 
 (defn custom-color-by-theme
+
   "(custom-color-by-theme color suffix-light suffix-dark opacity-light opacity-dark)
    color         :primary/:purple/...
    suffix-light  50/60
    suffix-dark   50/60
    opacity-light 0-100 (optional)
    opacity-dark  0-100 (optional)
    theme         :light/:dark (optional)"
+  {:deprecated
```

Comment:
Nice! I'd prefer if we were more direct, like in the Clojure style guide, example: 

```clojure
{:deprecated "Use `theme-colors` instead"}
```

Or since we don't version the code:

```clojure
{:deprecated true :superseded-by "theme/colors"}
```

The Clojure style guide also suggest using the docstring to explain in more detail why something is deprecated if we think there's a need for that.

Well, almost anything will do, but since we basically follow the CSG, why not for the deprecation metadata :)